specification/eventgrid/data-plane/Microsoft.ResourceNotifications/stable/2018-01-01/MaintenanceResources.json

```diff
@@ -0,0 +1,60 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-01-01",
+    "title": "Schema of Azure Resource Management events published to Azure Event Grid via Azure Resource Notifications",
+    "description": "Describes the schema of the Azure Resource Management events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent."
+  },
+  "paths": {},
+  "definitions": {
+    "MaintenanceResourcesScheduledEventEmittedEventData": {
```

Comment:
Are we planning to have more events that use ResourceInfoForScheduledEventEmittedEvent? If not, I would recommend inlining the properties here.